### ISSUE TITLE
character hiding

### ISSUE THREAD

**Issue by lightfirst** [jpcsp/jpcsp#407]
Kono Bushitsu wa Kitaku Shinai Bu ga Senkyo Shimashita. Portable: Gakuen Dog Ear Hen [ULJM06110]
Only find A little hair
![uljm06110-shot-1](https://cloud.githubusercontent.com/assets/4612168/13496305/680cb3fc-e18a-11e5-931a-16b365523b58.JPG)

**Comment by lightfirst** [comment]
[log.ZIP](https://github.com/jpcsp/jpcsp/files/157023/log.ZIP)

**Comment by lightfirst** [comment]
![Uploading ULJM06110-Shot-1.png…]()

**Comment by lightfirst** [comment]
PPSSPP show the whole character
![ppsspp_uljm06110_00000](https://cloud.githubusercontent.com/assets/4612168/13517951/d3365772-e203-11e5-818a-91ccb7fe7641.jpg)

**Comment by gid15** [comment]
How is the display using the external software renderer in Jpcsp?

**Comment by lightfirst** [comment]
![30](https://cloud.githubusercontent.com/assets/4612168/13814505/80ebbac6-ebc1-11e5-838e-14bafbc097c4.JPG)

**Comment by gid15** [comment]
OK.
The different seems to be in the dithering: the external software renderer is implementing it like on a real PSP. It is probably ignored in OpenGL rendering on PPSSPP as the PSP behavior can't be easily implemented using OpenGL.

**Comment by sum2012** [comment]
I don't have hiding problem without enable these option
(I am using Nvidia card with windows 7)
Enable saving GE screen to textures instead of memory
Enable dynamic shader generation (only relevant when using shaders, recommended for AMD/ATI)

**Comment by lightfirst** [comment]
Yes, If not enable saving GE screen to textures instead of memory, character will show
But there are some curved line on the face. And not high definition as PPSSPP
![31](https://cloud.githubusercontent.com/assets/4612168/13878300/62fbfa7c-ed4b-11e5-8e3d-ee7cfc4450c8.JPG)
